4-forms-validations/form.html

```diff
@@ -39,32 +39,32 @@ <h3>Personal Information</h3>
                 <br>
             <fieldset>
                 <h3>Socioeconomic status</h3>
-                <label>Stratum 1</label>
+                <label for="Stratum1">Stratum 1</label>
                 <input required type="radio" id="st1" value="1">
```

Comment:
```suggestion
                <label for="economicStatus1">Socioeconomic status 1</label>
                <input required type="radio" id="economicStatus1" value="1">
```

Reply:
Remember to add the `name` attribute. Without it you can select some options. We want to select only one option.